build.sh

```diff
@@ -16,6 +16,10 @@ REPOROOT="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 export CONFIGURATION="Debug"
 
 source "$REPOROOT/scripts/common/_prettyprint.sh"
+export TEMPLATES_BUILD=true
+export ENGINE_BUILD=true
+export CI_BUILD=false
+export SKIP_TESTS=false
 
```

Comment:
This is slightly different than the setup in build.ps1, specifically 
```
[bool]$TemplatesBuild=-not $CIBuild, 
[bool]$EngineBuild=-not $CIBuild, 
```
Not sure if that's an issue, probably not.